docs/content/docs/core-concept/transform.md

```diff
@@ -153,6 +153,23 @@ Flink CDC uses [Calcite](https://calcite.apache.org/) to parse expressions and [
 | COALESCE(value1 [, value2]*) | coalesce(Object... objects) | Returns the first argument that is not NULL.If all arguments are NULL, it returns NULL as well. The return type is the least restrictive, common type of all of its arguments. The return type is nullable if all arguments are nullable as well. |
 | IF(condition, true_value, false_value)   | condition ? true_value : false_value | Returns the true_value if condition is met, otherwise false_value. E.g., IF(5 > 3, 5, 3) returns 5. |
 
+## Casting Functions
+
+You can use `CAST( <EXPR> AS <T> )` syntax to convert any valid expression `<EXPR>` to a specific type `<T>`. Possible conversion paths are:
+
+| Source Type                         | Target Type | Notes                                                                                      |
+|-------------------------------------|-------------|--------------------------------------------------------------------------------------------|
+| ANY                                 | STRING      | All types can be cast to STRING.                                                           |
+| NUMERIC, STRING                     | BOOLEAN     | Any non-zero numerics will be evaluated to `TRUE`.                                         |
```

Comment:
Will -1 be treated as true?

Reply:
Yes. Any non-zero numeric will implicitly converts to `true` by C conventions.